changelogs/unreleased/6018-nested-status-properties.yml

```diff
@@ -0,0 +1,6 @@
+description: Support nested status properties in the Status Page
+issue-nr: 6018
+change-type: patch
+destination-branches: [master, iso7]
```

Comment:
Did you test it on iso7? Just for good measure.

Reply:
Yes, I run it on iso7 and it worked